godot-core/src/obj/onready.rs

```diff
@@ -65,10 +75,77 @@ use std::mem;
 ///        assert_eq!(*self.manual, 22);
 ///     }
 /// }
+/// ```
+///
+/// # Example - Macro-generated `init`
+/// ```
+/// use godot::prelude::*;
+///
+/// #[derive(GodotClass)]
+/// #[class(init, base = Node)]
+/// struct MyClass {
+///    base: Base<Node>,
+///    #[init(default = OnReady::new(|| 11))]
+///    auto: OnReady<i32>,
+///    #[init(default = OnReady::manual())]
+///    manual: OnReady<i32>,
```

Comment:
I'm not sure if it's a style we should showcase, since user-defined `init()` is likely the better choice here.

Maybe only show the `OnReady::manual()` and remove the `new()` one?